### DISCUSSION TITLE
Prebuilt VM images for JAX on GCP

### DISCUSSION THREAD

**Discussion by AakashKumarNain** [jax-ml/jax#16318] [General]
I was trying to do install JAX with GPU support on a linux machine. I followed the instructions provided [here](https://github.com/google/jax#installation) but `conda` install complains about the `glibc` version. Upgrading `glibc` *can* break a lot of things on the system. 

Given that there are VMs for TF, and PyTorch on GCP, are there any plans to provide VMs for JAX?

**Comment by ayaka14732** [comment]
To address your concern, it might be more beneficial to use `pip` for installation rather than `conda`. Pip, as the Python package manager, is typically more suited for managing Python packages.

Opting for a virtual machine might not be the optimal solution. It would be suggested to follow the installation guide provided on the JAX GitHub page: https://github.com/google/jax#installation. This guide allows the installation of the latest version of JAX using `pip`. After setting the version, the same command can be used on other machines to ensure version consistency.

While the convenience of prepackaged tools or images can be appealing, it might also introduce additional complications. Understanding and learning the correct usage of these tools could greatly enhance problem-solving efficiency in similar situations in the future.
